test/noInputRenameRule.spec.ts

```diff
@@ -111,6 +111,40 @@ describe(ruleName, () => {
           source
         });
       });
+
+      it("should fail when an input alias is prefixed by directive's selector, but the suffix does not match the property name", () => {
```

Comment:
Would you add specs for attribute selectors?

Reply:
Hi @mgechev I'm really sry for delay... but right now I can't do this because I'm traveling...

Reply:
No worries